modules/ytp-assets-common/src/less/components/dataset.less

```diff
@@ -28,3 +28,58 @@
   vertical-align: middle;
   height: 1.3em;
 }
+
+.admin-banner {
+  min-height: 40px;
+
+  a {
+    float: right;
+    color: rgb(42, 110, 187);
+    height: 40px;
+    font-size: 14px;
+    font-family: @font-family-sans-serif-semibold;
+    font-weight: @font-weight-heavier;
+    letter-spacing: 0.4px;
+    line-height: 20px;
+    border: 1px solid rgb(42, 110, 187);
+    border-radius: 2px;
+    padding: 10px 15px 0 15px;
+    text-decoration: none;
+  }
+}
+
+.private-or-draft-dataset {
+  background: #00347a;
```

Comment:
This is so close to variable `@suomi-brandBase` that maybe color here (or in variable definition) isn't correct and that variable should be used in here 🤔